cmd/trivy/main.go

```diff
@@ -105,6 +105,11 @@ OPTIONS:
 			Name:  "cache-dir",
 			Usage: "cache directory",
 		},
+		cli.StringFlag{
+			Name:  "vuln-type",
+			Value: "os,library",
+			Usage: "comma-separated list of vulnerability types (os, library)",
```

Comment:
`s/(os, library)/(os,library)/`
The space between "os" and "library" should be removed because the later code doesn't trim a space.